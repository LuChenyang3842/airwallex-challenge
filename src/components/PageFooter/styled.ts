import { Layout } from 'antd';
import styled from '@emotion/styled';
import { GLOABL_SIZE, THEME_COLOR } from 'src/constants/styleConstants';

export const Footer = styled(Layout.Footer)`
  width: 100vw;
  height: ${GLOABL_SIZE.footerHeight}px;
  background-color: ${THEME_COLOR.primaryLight};
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
`;
