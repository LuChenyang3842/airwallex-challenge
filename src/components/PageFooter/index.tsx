import { FC } from 'react';
import { Footer } from './styled';

const PageFooter: FC = () => {
  return <Footer></Footer>;
};

export default PageFooter;
