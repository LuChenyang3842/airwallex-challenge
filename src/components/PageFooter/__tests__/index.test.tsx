import { render } from '@testing-library/react';
import PageFooter from '../index';

it('PageFooter renders correctly', () => {
  const { baseElement } = render(<PageFooter />);
  expect(baseElement).toMatchSnapshot();
});
