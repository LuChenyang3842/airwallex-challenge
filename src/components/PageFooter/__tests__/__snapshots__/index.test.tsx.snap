// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter renders correctly 1`] = `
<body>
  <div>
    <footer
      class="ant-layout-footer css-1bowpb8"
    >
      <div
        class="css-1ysqs5i"
      >
        Made with ♥ in Melbourne 
      </div>
      <div
        class="css-1ysqs5i"
      >
        © 2016 Broccoli & Co. All Rights Reserved.
         
      </div>
    </footer>
  </div>
</body>
`;
